project-base/app/src/Model/Security/LoginAsUserFacade.php

```diff
@@ -99,4 +100,18 @@ public function getCurrentAdministratorLoggedAsCustomer(): ?Administrator
 
         return $this->administratorFacade->findByUuid($administratorUuid);
     }
+
+    /**
+     * @param \App\Model\Customer\User\CustomerUser $customerUser
+     * @return array{accessToken: string, refreshToken: string}
+     */
+    public function loginAndReturnAccessAndRefreshToken(CustomerUser $customerUser): array
```

Comment:
it should not be in project-base :slightly_smiling_face: Moreover, I do not feel like putting this functionality into the existing `LoginAsUserFacade` - the class is used when you need to login administrator as a user. IMHO it is worth creating a separate class for your purpose. But we can discuss it if you want

Reply:
fixed in https://github.com/shopsys/shopsys/pull/3154/commits/313cf3c2bb1afc6b35e6397eafc355c750391e37 as we agreed